gpu/README.md

```diff
@@ -200,13 +200,63 @@ spec:
           env:
             # add this env var, if using operator version 1.14.x
             - name: DD_ENABLE_NVML_DETECTION
-              value: "true" 
+              value: "true"
             # add this env var, if using operator versions 1.14.x or 1.15.x
             - name: DD_COLLECT_GPU_TAGS
-              value: "true" 
+              value: "true"
 ```
 
-For **mixed environments**, use the [DatadogAgentProfiles feature](https://github.com/DataDog/datadog-operator/blob/main/docs/datadog_agent_profiles.md) of the operator, which allows different configurations to be deployed for different nodes. In this case, it is not necessary to modify the DatadogAgent manifest. Instead, create a profile that enables the configuration on GPU nodes only:
+For **mixed environments**, use the [DatadogAgentProfiles (DAP) feature](https://github.com/DataDog/datadog-operator/blob/main/docs/datadog_agent_profiles.md) of the operator, which allows different configurations to be deployed for different nodes. Note that this feature is disabled by default, so it needs to be enabled. For more information, see [Enabling DatadogAgentProfiles](https://github.com/DataDog/datadog-operator/blob/main/docs/datadog_agent_profiles.md#enabling-datadogagentprofiles).
+
+Modifying the DatadogAgent manifest is necessary to enable certain features that are not supported by the DAP yet:
```

Comment:
```suggestion
Additionally, modify the DatadogAgent manifest to enable certain features that are not supported by the DAP yet:
```
☝🏻 Suggested wording to try to make these sections flow better together. Also, I noticed that we use both the phrases `the DatadogAgent manifest` and `the DatadogAgent configuration` - do they refer to the same thing? If so, it probably makes sense to use consistent terminology instead.